### PR TITLE
Tapi/config file

### PR DESCRIPTION
**Pull request by tapisky**
This PR will add the following features:
- Config file to be able to set all parameters to date (yaml)
- API keys and all other parameters can be set in user config file which is not part of the repo for convenience
- Provided pairs and coins parameters are checked for existence before script runs
- Provided "mechanism" to run script once or "forever" with given amount of seconds (set in config file) between iterations

### CODE REVIEW

File: corecito.py [hunk 1]
```diff
@@ -2,118 +2,166 @@
...
+
+    # Loop end
+    print(f'------------ Iteration {iteration} ------------\n')
+    if config['test_mode_on']:
+      await asyncio.sleep(1)
+      break
+    else:
+      # Wait given seconds until next poll
+      logger.info("Waiting for next iteration... ({} seconds)\n\n\n".format(config['seconds_between_iterations']))
+      await asyncio.sleep(config['seconds_between_iterations'])
```
Comment (sebaslogen) [inline corecito.py:102]: 💯 Awesome that you added the automatic iterations 😸

File: README.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# Corecito
+
+## Setting up
+
+*python3
+*To run this project, you may need to install the following python modules:
+
+```
+pip install cryptocom.exchange
+pip install import yaml
```
Comment (sebaslogen) [inline README.md]: This one didn't work completely for my, only the "import" library was installed. I got: 
```
$ pip3 install import yaml
Collecting import
  Using cached import-0.0.1.tar.gz (1.0 kB)
ERROR: Could not find a version that satisfies the requirement yaml (from versions: none)
ERROR: No matching distribution found for yaml
```

File: corecito.py [hunk 3]
```diff
@@ -2,118 +2,166 @@
...
-
-DEFAULT_CORE_NUMBER = 2 # Ethers
-MAX_DECIMALS_BUY = 6 # This is the limit Crypto.com has on ETH_BTC pair https://crypto.com/exchange/trade/spot/ETH_BTC
-MAX_DECIMALS_SELL = 3 # This is the limit Crypto.com has on ETH_BTC pair https://crypto.com/exchange/trade/spot/ETH_BTC
+# CRYPTOCOM_API_KEY = "MY_KEY"
+# CRYPTOCOM_API_SECRET = "MY_SECRET"
+#
+# DEFAULT_CORE_NUMBER = 2 # Ethers
+# config['max_decimals_buy'] = 6 # This is the limit Crypto.com has on ETH_BTC pair https://crypto.com/exchange/trade/spot/ETH_BTC
+# config['max_decimals_sell'] = 3 # This is the limit Crypto.com has on ETH_BTC pair https://crypto.com/exchange/trade/spot/ETH_BTC
```
Comment (sebaslogen) [inline corecito.py]: Your improvements made this code obsolete so we don't need it anymore. Would you like to remove it?

File: corecito.py [hunk 4]
```diff
@@ -2,118 +2,166 @@
...
-  max_core_number_decrease = core_number * 0.90 # <10% decrease
+def coreNumberDecreased(core_number, deviated_core_number, min_core_number_decrease_percentage, max_core_number_decrease_percentage):
+  min_core_number_decrease = core_number * (1 - (min_core_number_decrease_percentage/100))
+  max_core_number_decrease = core_number * (1 - (max_core_number_decrease_percentage/100))
   return deviated_core_number <= min_core_number_decrease and deviated_core_number >= max_core_number_decrease
 
-def coreNumberPlummeted(core_number, deviated_core_number):
-  max_core_number_decrease = core_number * 0.90 # >10% decrease
+def coreNumberPlummeted(core_number, deviated_core_number, max_core_number_decrease_percentage):
+  max_core_number_decrease = core_number * (1 - (max_core_number_decrease_percentage/100))
```
Comment (sebaslogen) [inline corecito.py:152]: Nice improvement & clean-up of duplicated values 👍

**Review by sebaslogen — APPROVED** [review]
Genial tio, muy buenos cambios, hay un par de comentarios para limpiar este PR y le peudes dar al boton de Merge PR

File: README.md (hunk 2 above)
Comment (tapisky) [reply README.md]: Oops, it was a copy/paste mistake. I have updated it already as it should be:

`pip3 install yaml`

**Pull request merged**